Gemfile

```diff
@@ -64,3 +64,5 @@ end
 gem 'tzinfo-data', platforms: %i[mingw mswin x64_mingw jruby]
 
 gem 'omniauth-gitlab', '~> 2.0'
+
+gem 'rest-client', '~> 2.1'
```

Comment:
Good choice! 👍 

We might have to Faraday later though

Reply:
I saw them both, and thought there isn't much happening as of now. People vouched for Faraday where there is a need of more fine grade control.